packit_service/worker/helpers/fedora_ci.py

```diff
@@ -28,8 +29,13 @@ def __init__(
     @property
     def target_branch(self) -> str:
         return (
-            self.metadata.event_dict.get("target_branch")  # for pagure.pr.Action
-            or self.metadata.event_dict.get("target")  # for koji.result.Task
+            # for pagure.pr.Comment
+            self.project.get_pr(self.metadata.pr_id).target_branch
```

Comment:
I see some overlap here with `dist_git_branch` property in the handler, could this be extracted?

Reply:
Let me check.